app/src/component/uriBar/view.js

```diff
@@ -32,17 +32,29 @@ class UriBar extends React.PureComponent {
       changeTextTimeout: null,
       currentValue: null,
       inputText: null,
-      focused: false
+      focused: false,
+      // TODO: Add a setting to enable / disable direct search?
+      directSearch: true
     };
   }
 
   handleChangeText = text => {
     const newValue = text ? text : '';
     clearTimeout(this.state.changeTextTimeout);
-    const { updateSearchQuery } = this.props;
+    const { updateSearchQuery, onSearchSubmitted, navigation } = this.props;
 
     let timeout = setTimeout(() => {
       updateSearchQuery(text);
+
+      if (!text.startsWith('lbry://')) {
```

Comment:
I would suggest having `internal-apis` handle this logic, eventually.

This can allow for a direct hit to show at top with related content shown below.

Reply:
This is here because we still want users to be able to directly navigate to lbry URLs instead of being just a plain search bar.